features/project/project_controller.rb

```diff
@@ -70,11 +70,32 @@ class ProjectController < AuthenticatedControllerBase
       erb(:new_project, locals: locals, layout: FastlaneCI.default_layout)
     end
 
+    get "#{HOME}/lanes/*/*/*" do |org, repo_name, branch|
```

Comment:
What is this URL used for?

Reply:
Maybe we can add a comment here with a little more details

Reply:
That's to be done once #360 is merged and I can actually test that everything works fine 😄